test/test_integration_cli_lambda.py

```diff
@@ -0,0 +1,234 @@
+import unittest
+import json
+
+from collections import namedtuple
+from io import TextIOWrapper
+from datetime import datetime
+from cdflow_commands import cli
+from mock import Mock, MagicMock, patch, mock_open, ANY
+
+
+class TestReleaseCLI(unittest.TestCase):
+
+    @patch('cdflow_commands.plugins.aws_lambda.ZipFile')
+    @patch('cdflow_commands.config.Session')
+    @patch('cdflow_commands.cli.Session')
+    @patch('cdflow_commands.plugins.aws_lambda.os')
+    @patch('cdflow_commands.config.open', new_callable=mock_open, create=True)
+    def test_release_package_is_created(
+        self, mock_open, mock_os,
+        session_from_cli, session_from_config, zip_file
+    ):
+        mock_metadata_file = MagicMock(spec=TextIOWrapper)
```

Comment:
It would be nice to add `# Given`, `# When` and `# Then` comments here to separate the individual logic within the test; would make it easier to read - at least to me